server/models/profile.js

```diff
@@ -0,0 +1,56 @@
+const mongoose = require("mongoose");
+const avaliableTime = new mongoose.Schema({
+    avaliableDay: {
+        type: Date,
+        required: true,
+    },
+    timeTo: {
+        type: String,
+        required: true,
+    },
+    timeFrom: {
+        type: String,
+        required: true
+    },
+})
+const profileSchema = new mongoose.Schema({
```

Comment:
We can use an 'enum' like object and add the `gender` field to this schema too 😄

Reply:
Done the changes